tools/grpcurl-query-ingesters/download-chunks-from-ingesters.sh

```diff
@@ -7,7 +7,10 @@ K8S_NAMESPACE=""
 MIMIR_TENANT_ID=""
 # End of configuration.
 
-mkdir -p chunks-dump/
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
```

Comment:
What I have here is what I usually use (for reasons I've long since forgotten... I think there was some edge case to do with symlinks or something like that).

If we use that pattern in other places, I'll just follow that for consistency.

Reply:
Resolved in 3abf2ce